dialogue-core/src/main/java/com/palantir/dialogue/core/ConcurrencyLimitedChannel.java

```diff
@@ -178,4 +186,9 @@ public String channelNameForLogging() {
             return channelNameForLogging;
         }
     }
+
+    @Value.Immutable
+    interface ConcurrencyLimitedChannelState {
+        CautiousIncreaseAggressiveDecreaseConcurrencyLimiter hostLimiter();
+    }
```

Comment:
I think we can put the `CautiousIncreaseAggressiveDecreaseConcurrencyLimiter` directly into the state, since it's the only piece of data we're tracking

Reply:
done